source/mainnet/net/references/multi-sig.rst

```diff
@@ -0,0 +1,448 @@
+
+   .. code-block:: json
+      :force:
+
+      {
+         "cidx": {
+            "kidx": {
+               "encryptedSignKey": {
+                  "metadata": {
+                     "encryptionMethod": "AES-256",
+                     "iterations": ...,
+                     "salt": ...,
+                     "initializationVector": ...,
+                     "keyDerivationMethod": "PBKDF2WithHmacSHA256"
+                  },
+                  "cipherText": ...
+               },
+               "verifyKey": ...,
+               "schemeId": "Ed25519"
+            },
+            ...
+         },
+         ...
+      }
+
+.. note::
+
+   These commands update the keys in the key directory of your local ``concordium-client`` tool. No transaction
+   is sent on-chain.
+
+
+Create a multi-sig transaction
+==============================
+
+Multi-sig transactions can be created for all active transaction types which are currently: ``DeployModule``,
+``InitContract``, ``UpdateContract``, ``Transfer(CCD)``, ``TransferWithMemo``, ``TransferWithSchedule``,
+``TransferWithScheduleAndMemo``, ``ConfigureDelegation``, ``ConfigureBaker``, ``RegisterData``,
+``UpdateCredential``, and ``UpdateCredentialsKeys``.
+
+To create a multi-sig transaction, execute a transaction-creating command and add an ``--out`` flag.
+This flag will cause ``concordium-client`` to output the multi-sig
+transaction into the specified file instead of sending it on-chain.
+
+If you omit the ``--signers`` flag, ``concordium-client`` will output
+a transaction signed with all keys associated with the account as present in the local key directory.
+If you want to sign with specific keys from the local key directory,
+you can specify some of them with the ``--signers`` flag (e.g. ``--signer "0:0,0:1"``).
+
+For example, to create a multi-sig transaction to send 1 CCD to an account on testnet using all local keys,
+run the following command:
+
+.. code-block:: console
+
+   $concordium-client transaction send --receiver 4bbdAUCDK2D6cUvUeprGr4FaSaHXKuYmYVjyCa4bXSCu3NUXzA --amount 1 --out ./transaction.json --energy 5000 --sender 4jxvYasaPncfmCFCLZCvuL5cZuvR5HAQezCHZH7ZA7AGsRYpix --grpc-port 20000 --grpc-ip node.testnet.concordium.com
+
+The multi-sig transaction is outputted into the ``transaction.json`` file and has the following format:
+
+.. code-block:: json
+   :force:
+
+   {
+      "energy": 5000,
+      "expiryTime": 1716995242,
+      "nonce": 46,
+      "payload": {
+         "amount": "1000000",
+         "toAddress": "4bbdAUCDK2D6cUvUeprGr4FaSaHXKuYmYVjyCa4bXSCu3NUXzA",
+         "transactionType": "transfer"
+      },
+      "signature": {
+         "0": {
+            "0": "3099534c5f32daf64dc40b7a0013979b9b74b167d259fc787a363ed2db7f1bcdafdcf06e166b2d915c7f29043186b3015a6064755bf3c3733bca2151b2b19c04"
+         }
+      },
+      "signer": "4jxvYasaPncfmCFCLZCvuL5cZuvR5HAQezCHZH7ZA7AGsRYpix",
+      "version": 1
+   }
```

Comment:
Should state somewhere that the transaction is created for a basic account with only one set of keys. I was kind of expecting multiple signatures as the paragraph just before stated "using all local keys" combined with the fact that we're describing multi-sig accounts/transactions